Modern/wrapperPlugins/src/main/java/org/bonej/wrapperPlugins/AnisotropyWrapper.java

```diff
@@ -302,12 +302,38 @@ private void addResults(final List<Subspace<BitType>> subspaces,
 		}
 	}
 
+	@SuppressWarnings("unused")
+	private void initializeIncrement() {
+		/* TODO: Get calibration from inputimage.axis(int)
+		   NB: you can't assume that 0, 1, 2 are X, Y, Z axes!
+		   NB: axes can have different units of calibration
+		 */
+		final double px = 1.0;
+		final double py = 1.0;
+		final double pz = 1.0;
+		final double diagonal = px * px + py * py + pz * pz;
+		// Round to 2 decimal places
+		minIncrement = Math.round(Math.sqrt(diagonal) * 100.0) / 100.0;
```

Comment:
This is OK only if uncalibrated distances are used for sampling, i.e. all sampling is based off a unit pixel grid and not pixel spacing in real units.

Reply:
Hence the comment in the code and in the PR. Anisotropy doesn't handle calibration yet, but that should be a different future PR.